packages/compiler/test/tsx-sourcemaps/template-windows.ts

```diff
@@ -0,0 +1,123 @@
+import { test } from 'uvu';
+import * as assert from 'uvu/assert';
+import { testSourcemap } from '../utils';
+
+test('template expression basic', async () => {
+  const input = `<div>{\r\nnonexistent\r\n}</div>`;
+
+  const output = await testSourcemap(input, 'nonexistent');
+  assert.equal(output, {
+    source: 'index.astro',
+    line: 2,
+    column: 1,
+    name: null,
+  });
+});
+
+test('template expression has dot', async () => {
+  const input = `<div>{\nconsole.log(hey)\n}</div>`;
```

Comment:
This test still uses `\n`, is this intended?

Reply:
Yes! Checking that the outputs are the same.